src/librustc_mir/transform/mod.rs

```diff
@@ -161,7 +161,7 @@ pub macro run_passes($tcx:ident, $mir:ident, $def_id:ident, $suite_index:expr; $
             promoted
         };
         let mut index = 0;
-        let mut run_pass = |pass: &dyn MirPassPassPass| {
+        let mut run_pass = |pass: &dyn MirPass| {
```

Comment:
what's up with these fixups?

Reply:
Macros are involved and rustfix broke. There's a bug filed for this.